index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Block-Chainference</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="style_d.css">
+  <script src="main.js" defer></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+
+<!--    Page for block chain coference 2023  -->
+
+<body>
+  <!--    bar menu and logo  -->
+  <header >
+    <section class="nav-bar-grey">
+      <a href="" class="nav-bar-grey-fabook"></a>
+      <a href="" class="nav-bar-grey-twitter"></a>
+      <a href="" class="nav-bar-grey-language">Language</a>
+      <a href="" class="nav-bar-grey-mypage">My Page</a>
+      <a href="" class="nav-bar-grey-logout">Logout</a>
+    </section>
+    <div class="nav-bar">
+      <section class="nav-menuContainer">
+        
+        <button class="bugerBtn">
+          <div class="burger-icon line1"></div>
+          <div class="burger-icon line2"></div>
+          <div class="burger-icon line3"></div>
+        </button>
+        <!--burger menu-->
+        <ul class="burger-menu">
+          <div class="burger-menu-style">
+              <li class="burger-menu-exit-button"><button class="exitButton"></button></li>
+              <li class="burger-menu-homePage-button"><a href="#home-page" class="burger-menu-home-page">Home Page</a></li>
+              <li class="burger-menu-aboutPage-button"><a href="#about-page" class="burger-menu-aboutPage-button">About</a></li>           
```

Comment:
- [ ] For both `Desktop, mobile` in the home and about, kindly refactor the header section using the `nav` tag to insert the social links, logo, and other navigation links. 
- This is an industry best practice which has to be followed to make you a better developer 👍